elements/lisk-api-client/src/codec.ts

```diff
@@ -189,7 +189,7 @@ export const decodeAssets = (
 	for (const asset of assets) {
 		const assetSchema = getAssetDataSchema(
 			blockVersion,
-			{ moduleID: asset.moduleID.toString('hex') },
+			{ module: asset.module.toString('hex') },
```

Comment:
`asset.module` is string, so it shouldnt need `toString`

Reply:
@sitetester You need to update BlockAsset and Event interfaces in `lisk-api-client` (module type from Buffer to string)